pkg/testutil/must.go

```diff
@@ -51,3 +51,18 @@ func MustWriteFile(filename string, data []byte, perm os.FileMode) {
 		panic(err)
 	}
 }
+
+// MustChdir simply wraps os.Chdir and panics if it fails.
```

Comment:
Describe what a function does, not what it is:

> MustChdir calls os.Chdir and panics if it fails.

Reply:
Done